docs/pii-encryption.md

```diff
@@ -0,0 +1,126 @@
+# login.gov PII Cryptography Overview
+
+`login.gov` is a service that provides single-sign-on capability to
+users of US government websites. It is developed jointly by 18F and USDS.
+Its goals include:
+
+* simplifying online interactions with the US government, by
+  reducing the number of usernames and passwords people need to remember;
+* improving security, by reducing the number of passwords people need to memorize
+  and implementing multi-factor authentication;
```

Comment:
s/implementing/requiring ? -- just to clarify that all user accounts require, at minimum, 2FA